Fix game not ending if last question wrong

cailafinn opened the pull request:
**Description of work.**
Added a checker to the question manager to see if all the questions have been answered.

**To test:**
Answer 9 questions right and get the last one wrong.

Fixes #132

Commit:
Fix game not ending if last question wrong RE #132

Alice-Jaffray reviewed (changes requested):
Works well, I answered 9 right and the last wrong - but should the 1 in the level select turn green after the case is completed?

ConorMFinn commented:
> Works well, I answered 9 right and the last wrong - but should the 1 in the level select turn green after the case is completed?

I figure it should. Sherry would have answered all the questions without getting sacked which I would consider a win. Otherwise it would put unfair pressure on the final question compared to the others

Commit:
Fix broken level select colours RE #132

Some extra buttons had the tags "LevelSelectButton" which was causing
the script to crash before it could change the colours.
In future, any buttons on the main menu should be checked to ensure they
do not have conflicting tags

Alice-Jaffray reviewed (approved):
looks good to me
